tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -583,6 +583,13 @@ test_i2c_iis2iclx: iis2iclx@6a {
 	int-pin = <1>;
 };
 
+test_i2c_hids: hids@5f {
+	compatible = "we,wsen-hids";
+	label = "WSEN-HIDS";
```

Comment:
remove the label property, its not needed.

Reply:
Done.